atd-vzv/src/App.css

```diff
@@ -16,3 +16,7 @@ a:focus {
   text-decoration: none;
   transition: all 0.3s;
 }
+
+div.Tooltip-module_tooltip__omIxd {
+  background: rgb(54, 51, 51) !important;
```

Comment:
is there a better way to do this without throwing an `!important` bomb at the problem? Does the reaviz docs have any tips on customizing the css?

![image](https://user-images.githubusercontent.com/5697474/71630063-a92c6780-2bc6-11ea-8696-72f2ae400e30.png)

Reply:
This one is a little more complicated! To answer your first question, I think the CSS selector is specific enough to not have to use !important (I removed it and the viz tooltip is still showing an opaque background). I scoured the documentation and tried many ways to customize the component's tooltip element using props to no avail, and when that failed I e-mailed the developer and never got a response :( I found the code that sets these parameters in the library and it seemed like they just weren't working properly (at least in this environment), so I asked @mddilley for help overriding the particular bit of styling and he figured out how to do it (thanks again, Mike)! I'll submit the next PR without the !important bomb. Let me know if that works or if you want me to try to find a more elegant solution.

Reply:
Yeah Mike and I talked about it too briefly. Bummer the library doesn't have better hook-ins for custom styling and getting the job done with the `!important` makes me feel better for sure. Good stuff.